mods/cnc/chrome/lobby-mappreview.yaml

```diff
@@ -36,6 +30,14 @@ Container@LOBBY_MAP_PREVIEW:
 					Height: 25
 					Font: Tiny
 					Align: Center
+				LabelWithTooltip@MAP_TITLE:		# MAP_TITLE is displayed above MAP_TYPE, MAP_AUTHOR, but to be in front of them for tooltip it was moved here below
```

Comment:
I see.  A cleaner simplest fix would be to add `IgnoreMouseOver: true` to the offending labels.